### DISCUSSION TITLE
Affiliates [Partnerships]

### DISCUSSION THREAD

**Discussion by MayiVT** [Paymenter/Paymenter#390] [Ideas]
Currently, Paymenter offers an affiliate system allowing users to generate their affiliate links and earn a percentage from sales. However, I propose integrating an exclusive Partnership feature to elevate the platform's potential for strategic partnerships. This feature would limit the creation of affiliate links to users with specific permissions, fostering deeper collaborations with content creators and other strategic partners.

**Comment by Valocai** [comment]
Amazing idea!

**Comment by VaibhavSys** [comment]
I have a feeling that this suggestion was written by an AI.

**Reply by MayiVT** [reply]
The grammar was corrected by one
